packages/react-examples/src/react/Layer/Layer.Customized.Example.tsx

```diff
@@ -1,66 +0,0 @@
-import * as React from 'react';
```

Comment:
It looks like @ThomasMichon originally added this example in #4518, so I'd be curious about his thoughts on its continued relevance (or not) before we remove it.

Reply:
I talked to Thomas. He was originally added because TMP has implemented this case. However we looked at TMP code and the use case is no longer there.

Regardless, I decided keep the example for now instead of removing it. I also updated the `Customizer` deprecation message to be more explicit.